kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloudconnection/Constants.java

```diff
@@ -0,0 +1,33 @@
+package org.eclipse.kura.cloudconnection;
+
+import org.eclipse.kura.cloudconnection.factory.CloudConnectionManagerFactory;
+
+/**
+ * @since 2.0
+ */
+public enum Constants {
```

Comment:
Doesn't it sound very generic? Couldn't we do something like `CloudConstants` or something similar?